rocAL_pybind/examples/parse_config.py

```diff
@@ -104,7 +104,7 @@ def parse_args():
         'python-unittest', 'python-unittest-related options')
     audio_unit_test.add_argument('--audio_path', type=str, default="",
                                   help='audio files path')
-    audio_unit_test.add_argument('--test_case', type=int, default=0,
```

Comment:
Can we add this in the audio 1st PR itself?

Reply:
Sure will do that